### PR TITLE
Create request

### PR DESCRIPTION
**Pull request by Ollesun**
CreateOffer -> CreateDeed,
EditOffer -> EditDeed,
diverse tester,
knsk ngn kommentar

### CODE REVIEW

File: goodDeeds/app/src/main/res/layout/edit_deed.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
         android:id="@+id/editOfferTextView"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="@string/edit_offer"
+        android:text="Edit Offer"
```
Comment (antjanne) [inline goodDeeds/app/src/main/res/layout/edit_deed.xml]: should use string resource <3

File: test.suml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Classdiagram><settings><option name="paintImplements" value="true" /><option name="paintDepends" value="false" /><option name="layoutOnChanges" value="false" /><option name="paintUses" value="false" /><option name="paintInner" value="true" /><option name="paintExtends" value="false" /></settings><classes /><stickycomponents /><textcomponents /><connectors /></Classdiagram>
```
Comment (antjanne) [inline test.suml]: What is this?
If you know, just ignore this comment

File: goodDeeds/app/src/main/res/layout/edit_deed.xml (hunk 1 above)
Comment (antjanne) [reply goodDeeds/app/src/main/res/layout/edit_deed.xml]: And good job btw 👍 🥇

File: test.suml (hunk 2 above)
Comment (Ollesun) [reply test.suml]: It's from SimpleUml. I thought I deleted it!

File: goodDeeds/app/src/main/res/layout/edit_deed.xml (hunk 1 above)
Comment (Ollesun) [reply goodDeeds/app/src/main/res/layout/edit_deed.xml]: :+1:

File: goodDeeds/app/src/test/java/com/goodpeople/gooddeeds/model/GoodDeedsTest.java [hunk 3]
```diff
@@ -215,5 +231,18 @@ public void deleteCurrentDeed() {
         Assert.assertFalse(gd.getIDeeds().contains(deed));
     }
 
+    @Test(expected = RuntimeException.class)
+    public void deleteCurrentDeedExcepiton() {
```
Comment (GeneralMustard) [inline goodDeeds/app/src/test/java/com/goodpeople/gooddeeds/model/GoodDeedsTest.java]: spelling

File: goodDeeds/app/src/test/java/com/goodpeople/gooddeeds/model/GoodDeedsTest.java [hunk 4]
```diff
@@ -187,6 +189,14 @@ public void claimDeed() {
...
     }
 
+    @Test
+    public void claimDeed2() {
+        gd.setCurrentdeed(gd.getIDeeds().get(1).getUuid());
+        gd.claimDeed();
+        assertTrue((gd.getCurrentDeed().getGivingAccount() != null) &&
+                (gd.getCurrentDeed().getReceivingAccount() != null));
+    }
+
```
Comment (GeneralMustard) [inline goodDeeds/app/src/test/java/com/goodpeople/gooddeeds/model/GoodDeedsTest.java]: better name?

**Pull request merged**